src/components/AdherenceCalendar.js

```diff
@@ -27,6 +27,11 @@ const Calendar = styled(ReactCalendar)`
     text-decoration: underline;
     font-size: 1.2rem;
   }
+
+  & button {
```

Comment:
I don't think this was just a Chrome problem. As my browser on Safari is still having troubles